build.assets/windows/build.ps1

```diff
@@ -444,6 +444,44 @@ function Build-Connect {
     Write-Host $("Built Teleport Connect in {0:g}" -f $CommandDuration)
 }
 
+function Write-Version-Objects {
+    [CmdlesBinding()]
+    param(
+        [Parameter(Mandatory)]
+        [string] $TeleportSourceDirectory,
+        [Parameter(Mandatory)]
+        [string] $TeleportVersion,
+    )
+    Write-Host "Generating version info files for tsh.exe and tctl.exe..."
+
+    # install go-winres (v0.3.3)
+    go install github.com/tc-hib/go-winres@d743268d7ea168077ddd443c4240562d4f5e8c3e
```

Comment:
IMO we need to capture this with automated dep updates somehow, especially given that it's used to modify executables before they're signed.

Reply:
Could we alternatively use this as a generator instead, and capture the dependency via go.mod?

Reply:
This does not modify executables in any way. It produces a resource file that the `go` tool uses to build a binary.

I'd argue that we _don't_ want automated updates on this. The current version does just what we need and no more. Blindly updating parts of the build toolchain is much riskier than pinning to a known-good version, no?